gitman/models/source.py

```diff
@@ -16,19 +16,21 @@
 @yorm.attr(rev=String)
 @yorm.attr(link=NullableString)
 @yorm.attr(scripts=List.of_type(String))
+@yorm.attr(sparse_paths=List.of_type(String))
```

Comment:
The order of these decorators determines the order of attributes in generated YAML files. I'd prefer the `sparse_paths` setting to be between `repo` and `rev`.

Reply:
Allright. Having `scripts` as last option makes sense, because it can grow quite large.

Reply:
Changed in the latest commit.